### PR TITLE
UG(900812): Updated assistview images in all platforms.

### PR DESCRIPTION
**Pull request by IndrajithSrinivasan**
Updated assistview with jpg images in all platforms.

### CODE REVIEW

**Review by silambarasani — APPROVED** [review]

File: javascript/javascript-ai-assistview.jpg
Comment (silambarasani) [inline javascript/javascript-ai-assistview.jpg:1]: Revert this change. It will break for JavaScript.
https://www.npmjs.com/package/@syncfusion/ej2-interactive-chat

**Review by silambarasani — CHANGES_REQUESTED** [review]

File: blazor/blazor-ai-assistview.jpg
Comment (silambarasani) [inline blazor/blazor-ai-assistview.jpg:1]: Revert this change. Break for nuget package
https://www.nuget.org/packages/Syncfusion.Blazor.InteractiveChat

**Review by silambarasani — CHANGES_REQUESTED** [review]

File: javascript/javascript-ai-assistview.jpg
Comment (IndrajithSrinivasan) [reply javascript/javascript-ai-assistview.jpg:1]: I haven't removed images @silambarasani , it will not break

image

File: blazor/blazor-ai-assistview.jpg
Comment (IndrajithSrinivasan) [reply blazor/blazor-ai-assistview.jpg:1]: I haven't removed images @silambarasani , it will not break. Added new image with jpg extension

image

**Review by silambarasani — APPROVED** [review]
